meeting-notes/20181127.md

```diff
@@ -0,0 +1,227 @@
+# Dynamic Data and Capabilities WG — November 27th, 2018
+
+- **Lead:** @pgte
+- **Notetaker:** @jimpick
+- **Attendees:**
+  - @pgte
+  - @jimpick
+  - @andreforsousa
+  - @daviddahl
+  - @arkadiy
+  - @aschmahmann
+  - @marcooliveira
+  - @satazor
+
+- [**Zoom.us meeting URL**](https://protocol.zoom.us/j/586782711)
+- [**Recording**](https://www.youtube.com/watch?v=CY_cVPEATc8&feature=youtu.be)
+
+## Agenda
+
+1. Start recording
+1. Ask everyone to put their name into the list of attendees
+1. Ask for a volunteer to take notes
+1. Everyone can add items to this agenda for things they would like to discuss
+1. Round of intros and updates
+  - Introduce yourself and your interest in this subject
+  - What have you accomplished since the last meeting?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+1. Demos
+  - peer-pad E2E load tests
+  - discussify replies
+1. Questions
+
+### 2 Week Updates:
+
+(please copy this and add your update here :)
+
+#### @
+ - Concluded:
+ - In progress:
+ - Blocked:
+ - Next:
+
+#### @satazor
+ - Concluded:
+   - Replies
+   - Lazy load of comments and replies (view more)
+ - In progress:
+   - versidag + ipfs-versidag modules
+     - https://github.com/ipfs/dynamic-data-and-capabilities/issues/50
+ - Blocked:
+   - Add persistance of the data (pinner + sync UI language)
+     - Infrastructure
+     - Sync UI language being done for peerpad; will later borrow the same UI language
+ - Next:
+   - Integrate ipfs-versidag into discussify so that we have history
+     - This will require a small refactor since some of stuff currently in the CRDT is no longer necessary
+   - Either launch discussify in chrome/firefox store
```

Comment:
Why not both?

Reply:
Kickstarting the Identity/IDM will require me to do a technical top-down analysis to decompose it in smaller parts. Those parts can be made individually and even in parallel. There's a few people that want to contribute and those smaller parts can be assigned to whoever is interested in helping.

There's important stuff that needs to be done for Discussify to be launched in the stores:

- Make manifest.json compatible with Firefox: this will require to setup a manifest.json template and run it for Chrome and Firefox
- Pinner must be working; it would be bad if data is simply lost
- Add versioning to the data-model so that we can do breaking changes if necessary
  - What we are thinking for now is simply suffixing the collaboration name with a version. It's assumed that whenever the version changes, all discussions are reset. Note that the data isn't actually lost, because you can always install the previous version to see the old comments.
  - Later, we should think carefully on how we deal with migrations. This is a use-case of many DApps and I think that it's worth discussing it as a feature of peer-star-app.

*The history & new header can be later added via an updated*

I'm afraid that if I dedicate the time to do what I listed above, I will only be able to kickstart the identity in 1w+. I've talked with @pgte about adding Pedro Santos to help launching Discussify with these simpler tasks so that I can focus on Identity. He will follow up with you on that via email.